src/string.cr

```diff
@@ -2260,7 +2260,7 @@ class String
   # returns the number of times the block returned a truthy value.
   #
   # ```
-  # "aabbcc".count { |c| ['a', 'b'].includes?(c) } # => 4
+  # "aabbcc".count(&.in?('a', 'b')) # => 4
```

Comment:
```suggestion
  # "aabbcc".count &.in?('a', 'b') # => 4
```

Usually the last block is used without parenthesis in other doc examples.
There are other ocurrences also of this in the PR

Reply:
I'm fine with either version.

Should I remove the parenthesis from all of the examples?
- 👍 — YES
- 👎 — NO

Reply:
Done.